nx.json

```diff
@@ -115,5 +117,6 @@
       }
     },
     "releaseTagPattern": "v{version}"
-  }
+  },
+  "useInferencePlugins": false
```

Comment:
also, just so you are aware i left the default configuration from Nx NOT to use project Crystal. You might want to opt in instead...

Reply:
Let's create a followup issue for it so we can assign it to someone later please.